.github/workflows/docker.yml

```diff
@@ -30,3 +30,26 @@ jobs:
         run: |
           bashbrew push --dry-run caddy
         if: github.repository != 'caddyserver/caddy-docker' || github.ref != 'refs/heads/master'
+  docker-windows-build:
+    runs-on: windows-2019
+    # env:
+    #   DOCKER_BUILDKIT: '1'
+    steps:
+      - uses: actions/checkout@master
+      - name: non-master build test
+        run: |
+          docker build -f windows/Dockerfile windows
+        if: github.repository != 'caddyserver/caddy-docker' || github.ref != 'refs/heads/master'
+      # - name: build
+      #   run: bashbrew build caddy
```

Comment:
Not sure if we can get this to run (easily) in Windows... I think it's enough to let the official build servers publish the windows flavour.